specification/cognitiveservices/data-plane/Personalizer/stable/v1.0/Personalizer.json

```diff
@@ -479,6 +479,80 @@
         }
       }
     },
+    "/metrics/learning": {
+      "get": {
+        "tags": [
+          "Metrics"
+        ],
+        "summary": "Get learning performance metrics.",
+        "description": "Get learning performance metrics aggregated by time or events.",
+        "operationId": "LearningMetrics_Get",
+      "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "StartDate",
+            "in": "query",
+            "description": "Start date to get the metrics for.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "EndDate",
+            "in": "query",
+            "description": "End date to get the metrics for.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "AggregationInterval",
```

Comment:
>AggregationInterval [](start = 21, length = 19)

Change to duration type. example: 01:00:00 for one hour.

Reply:
Review how other services are accepting time duration in the azure portal.

---
In reply to: [466661568](https://github.com/Azure/azure-rest-api-specs/pull/10366#discussion_r466661568) [](ancestors = 466661568)